CalcualtorLogic.js

```diff
@@ -106,8 +106,8 @@ class Calculator {
     onButtonNumberClick(event) {
         const currentResultAndclickedButton = this.currentResult + event.target.dataset.number;
         this.currentResult = currentResultAndclickedButton;
-        this.operator = event.target.dataset.operator;
-        this.updateResultElement();
+        this.updateResultElement(this.operator);
```

Comment:
can you please explain why you did this change

Reply:
I just tried to do something without knowing what I am doing 
maybe it will be true

especially this class calculator project is my first project 
still now I do not know how it will work